pkg/app/piped/controller/scheduler.go

```diff
@@ -661,6 +661,10 @@ func (s *scheduler) getMentionedAccounts(ctx context.Context, event model.Notifi
 		return nil, err
 	}
 
+	if ds.GenericDeploymentConfig.DeploymentNotification == nil {
+		return nil, nil
+	}
+
```

Comment:
nits
- should be done before line L654 to avoid unnecessary targetDSP fetch.
- add log (using s.logger) to show why does this return with no error is better.

Reply:
Or just having a comment works fine.

Reply:
Sorry, I can't understand the below one.

> should be done before line L654 to avoid unnecessary targetDSP fetch.

To check if `DeploymentNotification` is not nil, we need targetDSP fetch, right?

Reply:
@ono-max  Ah, that was my missed 🙏

Reply:
87ccd80